packages/rxsocketio/src/operators/send.js

```diff
@@ -37,4 +108,5 @@ const send = function send({
   };
 };
```

Comment:
I refactored the publicly exposed function so that it focuses on glueing logic together instead of encapsulating any business logic that would require testing -- since those pieces are easier to test as separate units.